owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt

```diff
@@ -225,6 +227,9 @@ class FileDisplayActivity : FileActivity(), FileFragment.ContainerActivity, OnEn
             createMinFragments()
         }
 
+        val dataIntent: Uri? = intent.data
+        manageDataIntent(dataIntent)
```

Comment:
In `manageDataIntent` we have several checks to ensure that the URI received is not null that could be removed in an easy way.

```suggestion
        dataIntent?.let { manageDataIntent(dataIntent) }
```

BTW, `manageDataIntent` is way too uncertain. Something like `handleDeepLink` could be more accurate, what do u think?

Reply:
Fixed